curl_cffi/aio.py

```diff
@@ -251,3 +210,45 @@ def set_exception(self, curl: Curl, exception):
     def setopt(self, option, value):
         """Wrapper around curl_multi_setopt."""
         return lib.curl_multi_setopt(self._curlm, option, value)
+
+
+@ffi.def_extern()
+def timer_function(curlm, timeout_ms: int, clientp: AsyncCurl):
```

Comment:
I don't quite get why these functions are moved in the end. Actually, we can annotate it as `"AsyncCurl"` even if it's put in the beginning.

Reply:
Done